src/panel-manager.js

```diff
@@ -86,6 +94,7 @@ genotet.panelManager.activatePanel_ = function(viewID) {
  * Creates a panel with the given name.
  * @param {!genotet.View} view Name of the view.
  * @return {!jQuery}
+ * @this
```

Comment:
I am sorry that I don't understand why we need to avoid using namespace as @this. Could you please explain it in detail? Thank you very much!